allure-cucumber/lib/allure_cucumber/formatter.rb

```diff
@@ -111,7 +111,7 @@ def handle_step_started(test_step)
     # @param [Cucumber::Core::Test::HookStep] hook_step
     # @return [void]
     def handle_hook_started(hook_step)
-      lifecycle.public_send(HOOK_HANDLERS[hook_step.text], cucumber_model.fixture_result(hook_step))
+      lifecycle.public_send(HOOK_HANDLERS[hook_step.text], cucumber_model.fixture_result(hook_step)) unless !HOOK_HANDLERS.key?(hook_step.text)
```

Comment:
It's probably better to use guard clause:

```ruby
return unless HOOK_HANDLERS.key?(hook_step.text)

lifecycle.public_send(HOOK_HANDLERS[hook_step.text], cucumber_model.fixture_result(hook_step))
```

Reply:
Yes, agree. I've modified it